CSharpExample/Class1.cs

```diff
@@ -1,17 +1,25 @@
 ﻿using System;
 using System.IO;
+using System.Threading;
```

Comment:
This just gets rid of the syntactic sugar and also stops including `new FileInfo`.